backend/corpora/common/utils/uuid.py

```diff
@@ -0,0 +1,5 @@
+import uuid
+
+
+def generate_uuid():
```

Comment:
nit -- does this need to be in its own file? can it go in db_utils? or as a function on the base of the orm?

Reply:
I could put it in ORM, but it doesn't need to be apart of base.